Source/Extensions/Blazorise.Splitter/Splitter.razor.cs

```diff
@@ -0,0 +1,184 @@
+                    DragInterval = DragInterval,
+                    Direction = Direction,
+                    Cursor = Cursor,
+                };
+
+                JSSplitInstance = await JSModule.InitializeSplit( splitSections, options );
+            }
+        }
+        finally
+        {
+            createInstanceLock.Release();
+        }
+    }
+
+    #endregion
+
+    #region Properties
+
+    /// <summary>
+    /// Gets or sets the <see cref="JSSplitModule"/> instance.
+    /// </summary>
+    protected JSSplitModule JSModule { get; private set; }
+
+    /// <summary>
+    /// Gets or sets the reference to the JS split instance
+    /// </summary>
+    protected IJSObjectReference JSSplitInstance { get; private set; }
+
+    /// <summary>
+    /// Gets or sets the JS runtime.
+    /// </summary>
+    [Inject] private IJSRuntime JSRuntime { get; set; }
+
+    /// <summary>
+    /// Gets or sets the version provider.
+    /// </summary>
+    [Inject] private IVersionProvider VersionProvider { get; set; }
+
+    /// <summary>
+    /// Initial sizes of each element in percents or CSS values.
+    /// </summary>
+    [Parameter] public IEnumerable<JavascriptNumber> Sizes { get; init; }
+
+    /// <summary>
+    /// Minimum size of each element.
+    /// </summary>
+    [Parameter] public JavascriptNumberOrArray MinSize { get; init; }
+
+    /// <summary>
+    /// Maximum size of each element.
+    /// </summary>
+    [Parameter] public JavascriptNumberOrArray MaxSize { get; init; }
+
+    /// <summary>
+    /// When the split is created, if ExpandToMin is true, the minSize for each element overrides the percentage value from the sizes option.
+    /// </summary>
+    [Parameter] public bool? ExpandToMin { get; init; }
+
+    /// <summary>
+    /// Gutter size in pixels. Defaults to 10.
+    /// </summary>
+    [Parameter] public JavascriptNumber GutterSize { get; init; } = 10;
+
+    /// <summary>
+    /// Determines how the gutter aligns between the two elements.
+    /// </summary>
+    [Parameter] public SplitGutterAlignment GutterAlign { get; init; } = SplitGutterAlignment.Center;
+
+    /// <summary>
+    /// Snap to minimum size offset in pixels.
+    /// </summary>
+    [Parameter] public JavascriptNumberOrArray SnapOffset { get; init; }
+
+    /// <summary>
+    /// Drag this number of pixels at a time. Defaults to 1 for smooth dragging, but can be set to a pixel value to give more control over the resulting sizes.
+    /// Works particularly well when the <see cref="GutterSize"/> is set to the same size.
+    /// </summary>
+    [Parameter] public JavascriptNumber DragInterval { get; init; }
+
+    /// <summary>
+    /// Direction to split.
+    /// </summary>
+    [Parameter] public SplitDirection Direction { get; init; } = SplitDirection.Horizontal;
+
+    /// <summary>
+    /// Cursor to display while dragging.
+    /// </summary>
+    [Parameter] public string Cursor { get; init; }
```

Comment:
It's hard to do with an enum because the whole idea is that the user can practically define any cursor name. And I don't want to have an enum for all of them. I think it would be better to leave it like this.